chaincode/compute_plan.go

```diff
@@ -413,20 +413,23 @@ func UpdateComputePlanState(db *LedgerDB, ComputePlanID, tupleStatus, tupleKey s
 	if err != nil {
 		return err
 	}
-	statusUpdated := cp.UpdateStatus(tupleStatus)
+	stateUpdated := cp.UpdateState(tupleStatus)
 	doneModels := []string{}
-	if tupleStatus == StatusDone {
-		doneModels, err = cp.CheckDoneIntermediaryModel(db)
+	if tupleStatus == StatusDone && cp.CleanModels {
```

Comment:
👍

Reply:
It's clearer but it's also checked at the fist line of `CheckDoneIntermediaryModel`

Reply:
This extra check is not all that useful, I could remove it

Reply:
Yes but at first read I thought it was useful even though I've wrote the other check myself... So as I said it's not useful but probably clearer. Therefore I've no strong opinion either way.